pages/dashboard/index.tsx

```diff
@@ -7,15 +7,81 @@ import nextI18nextConfig from 'next-i18next.config';
 import { serverSideTranslations } from 'next-i18next/serverSideTranslations';
 import { GetStaticProps } from 'next';
 import withAuth from '@/components/HOCs/withAuth';
+import Transaction from './Transaction';
+import { FiUser, FiFileText, FiDollarSign } from 'react-icons/fi';
+import { Chart as ChartJS, CategoryScale, LinearScale, PointElement, LineElement, Title, Tooltip, Legend } from 'chart.js';
+import { Line } from 'react-chartjs-2';
+
+ChartJS.register(CategoryScale, LinearScale, PointElement, LineElement, Title, Tooltip, Legend);
 
 type Props = {
   loading: boolean;
 };
+enum CheckReduceIncrease {
+  REDUCE,
+  INCREASE,
+  EQUAL,
+}
 
 const Dashboard = (props: Props) => {
   return (
     <MainLayout customClass="dashboard-page" headData={{ title: 'Dashboard' }}>
-      <h1 className="mx-6 text-3xl font-semibold">Dashboard</h1>
+      <h1 className="mx-6 mb-[34px] text-3xl font-semibold">Dashboard</h1>
+      <div className="mb-[48px] flex max-w-[1120px] justify-between space-x-[50px] ">
+        <Transaction Icon={FiUser} checkReduceIncrease={CheckReduceIncrease.INCREASE} price={23} name={'transaction'} value={1} />
+        <Transaction Icon={FiDollarSign} checkReduceIncrease={CheckReduceIncrease.REDUCE} price={23} name={'Total users'} value={1} />
+        <Transaction
+          Icon={FiFileText}
+          checkReduceIncrease={CheckReduceIncrease.INCREASE}
+          price={23}
```

Comment:
Mock data by creating a sample object in /utils. Because the data response from API will saved as a object